tests/bluechi_machine_lib/bluechi_is_online.py

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright Contributors to the Eclipse BlueChi project
+#
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+import unittest
+import subprocess
+from typing import Tuple
+
+
+class BluechiIsOnline:
+    """
+    A Python module for the 'bluechi-is-online' command to check the status of agents, nodes, and the system.
+    """
+
+    def __init__(self, executable_path: str = "/usr/bin/bluechi-is-online"):
+        self.executable_path = executable_path
+
+    def _run_command(self, args: list) -> Tuple[int, str, str]:
+        command = [self.executable_path] + args
+        try:
+            process = subprocess.Popen(
+                command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, text=True
+            )
+            print(f"Executing command: {process.args}")
+            stdout, stderr = process.communicate()
+
+            print(
+                f"Command execution completed: '{process.args}', "
+                f"return code: {process.returncode}, stdout: '{stdout.strip()}', stderr: '{stderr.strip()}'"
+            )
+
+            return process.returncode, stdout.strip(), stderr.strip()
+
+        except Exception as e:
+            print(f"Error while executing command '{command}': {e}")
+            raise
+
+
+class TestBluechiIsOnline(unittest.TestCase):
```

Comment:
Note that this shouldn't be a test, but just a library to reuse when running code in the bluechi container/vm.